addons/account/static/src/js/reconciliation/reconciliation_model.js

```diff
@@ -1027,6 +1054,13 @@ var StatementModel = BasicModel.extend({
                     return data.write_off ? self.quickCreateProposition(line.handle, data.model_id) : true;
                 })
             );
+            // If still no partner set, take the one from context, if it exists
```

Comment:
This seems wrong..
You probably wanted to put that part after the last 'then' 2 lines above.

```
.then(function(){
                    return data.write_off ? self.quickCreateProposition(line.handle, data.model_id) : true;
                })
.then(function(){
       if (!line.st_line.partner_id && self.context.partner_id && self.context.partner_name) {
                return self.changePartner(line.handle, {
                    'id': self.context.partner_id,
                    'display_name': self.context.partner_name,
                }, true);
            }
           return true;
})
```

Reply:
It seems to be doing the job just fine though, what would be the problem with this?